vignettes/esApply.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "esApply Introduction"
+author: "R. Gentleman, V. Carey, M. Morgan, S. Falcon"
```

Comment:
Here I think the format should be

```
author:
- name "R. Gentleman"
- name "V. Carey"
- name "M. Morgan"
- name "S. Falcon"
```
I think also that the translator deserves acknowledgement as an author with the role clearly specified. Adding the following would do this, although it is a 'hack' that re-uses the 'affiliation: ' tag (the affiliation appears as a footnote...)

```
- name: "H. Khan"
  affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
```